.editorconfig

```diff
@@ -545,7 +545,7 @@ dotnet_diagnostic.CA1720.severity = error
 dotnet_diagnostic.CA1810.severity = error
 dotnet_diagnostic.CA1821.severity = error
 dotnet_diagnostic.CA1822.severity = error
-dotnet_diagnostic.CA1823.severity = error
```

Comment:
Can you leave this as error?

Reply:
Setting to warning is appropriate for development experience. During any non-ide build, warnings are turned into errors. (see msbuild properties `TreatWarningsAsErrors` and `CodeAnalysisTreatWarningsAsErrors`) This is a recommended pattern that speeds up local development.

Reply:
won't fix